lib/schema.ts

```diff
@@ -100,32 +108,33 @@ function buildFields(schema, rLevelParentMax, dLevelParentMax, path) {
     }
 
     /* nested field */
+    
     if (opts.fields) {
       fieldList[name] = {
         name: name,
-        path: path.concat([name]),
+        path: path.concat(name),
         repetitionType: repetitionType,
         rLevelMax: rLevelMax,
         dLevelMax: dLevelMax,
         isNested: true,
-        statistics: opts.statistics,
+        // statistics: opts.statistics,
         fieldCount: Object.keys(opts.fields).length,
         fields: buildFields(
               opts.fields,
               rLevelMax,
               dLevelMax,
-              path.concat([name]))
+              path.concat(name))
       };
 
-      if (opts.type == 'LIST' || opts.type == 'MAP') fieldList[name].originalType = opts.type;
-
```

Comment:
LIST and MAP are not valid typings so this should always returns false. I removed it for now